pkg/controller/kieapp/constants/constants.go

```diff
@@ -260,6 +260,26 @@ const (
 	//Envs
 	KubeNS     = "KUBERNETES_NAMESPACE"
 	KubeLabels = "KUBERNETES_LABELS"
+
+	//Kafka extension
+	KIE_KAFKA_EXT_DISABLED         = "org.kie.kafka.server.ext.disabled"
```

Comment:
These envs that sets the properties would need to go on the image, yes?
@jakubschwan @tchughesiv wdyt?

Reply:
yes, looks like these should go to the image. I would left there only constants for default values if needed

Reply:
jboss-container-images/rhdm-7-openshift-image#413
jboss-container-images/rhpam-7-openshift-image#524